### ISSUE TITLE
existing package at remote http repo stops copy

### ISSUE THREAD

**Issue by e3krisztian** [e3krisztian/pyrene#1]
Upload throws a HTTP Conflict 409 error, when exactly the same version exists at the destination repo.

Unfortunately this is uncatched, so not all dependencies are copied.

**Comment by e3krisztian** [comment]
The exact error is:

```
    raise HTTPError(http_error_msg, response=self)
HTTPError: 409 Client Error: Conflict
```

from the `requests` package

**Comment by e3krisztian** [comment]
fix: ee230cbea17427f1d4f614d8a41c1aa394d71c8c
released in 0.1.3
